.github/workflows/build-publish-develop-pr.yml

```diff
@@ -40,67 +32,83 @@ jobs:
         with:
           ref: ${{ env.GIT_REF }}
 
-      # This gets the image tag and whether to publish the image based on the event type
-      # PR builds: pr-<pr_number>-<short_sha> (if label 'build-publish' is present publishes the image)
-      # develop builds: develop-<short_sha> and develop (only amd64)
-      # release builds: release-<short_sha>
-      # manual builds: <short_sha> (if build-publish is true publishes the image)
-      - name: Get image tag
-        id: get-image-tag
-        run: |
-          short_sha=$(git rev-parse --short HEAD)
-          echo "build-publish=false" | tee -a $GITHUB_OUTPUT
-          if [[ ${{ github.event_name }} == 'push' ]]; then
-            if [[ ${{ github.ref_name }} == 'release/'* ]]; then
-              echo "image-tag=release-${short_sha}" | tee -a $GITHUB_OUTPUT
-              echo "build-publish=true" | tee -a $GITHUB_OUTPUT
-            else
-              echo "image-tag=develop" | tee -a $GITHUB_OUTPUT
-              echo "build-publish=true" | tee -a $GITHUB_OUTPUT
-            fi
-          elif [[ ${{ github.event_name }} == 'workflow_dispatch' ]]; then
-            echo "image-tag=${short_sha}" | tee -a $GITHUB_OUTPUT
-            echo "build-publish=${{ github.event.inputs.build-publish }}" | tee -a $GITHUB_OUTPUT
-          else
-            if [[ ${{ github.event_name }} == "pull_request" ]]; then
-              echo "image-tag=pr-${{ github.event.number }}-${short_sha}" | tee -a $GITHUB_OUTPUT
-              if [[ ${{ contains(github.event.pull_request.labels.*.name, 'build-publish') }} == "true" ]]; then
-                echo "build-publish=true" | tee -a $GITHUB_OUTPUT
-              fi
-            fi
-          fi
-
       - name: Configure aws credentials
-        if: steps.get-image-tag.outputs.build-publish == 'true'
         uses: aws-actions/configure-aws-credentials@e3dd6a429d7300a6a4c196c26e071d42e0343502 # v4.0.2
         with:
           role-to-assume: ${{ secrets.AWS_OIDC_IAM_ROLE_BUILD_PUBLISH_DEVELOP_PR }}
           aws-region: ${{ secrets.AWS_REGION }}
           mask-aws-account-id: true
-          role-session-name: goreleaser-build-publish-${{ matrix.image-name }}
+          role-session-name: "merge"
+
+      - uses: actions/cache/restore@v4
+        with:
+          path: dist/linux_amd64_v1
+          key: chainlink-amd64-${{ github.sha }}
+          fail-on-cache-miss: true
+
+      - uses: actions/cache/restore@v4
+        with:
+          path: dist/linux_arm64
```

Comment:
Why does `amd64` have `_v1`, and `arm64` does not?

Reply:
https://goreleaser.com/customization/builds/#why-is-there-a-_v1-suffix-on-amd64-builds